packages/client-twitter/src/post.ts

```diff
@@ -486,7 +486,7 @@
             const removeQuotes = (str: string) =>
                 str.replace(/^['"](.*)['"]$/, "$1");
 
-            const fixNewLines = (str: string) => str.replaceAll(/\\n/g, "\n");
+            const fixNewLines = (str: string) => str.replaceAll(/\\n/g, "\n\n"); //ensures double spaces
```

Comment:
Modified the fixNewLines function to ensure double line breaks are applied, enhancing tweet readability.